crates/test_utils/wasm/wasm_workspace/link/src/integrity.rs

```diff
@@ -56,17 +58,20 @@ pub fn validate_create_link_by_must_get_agent_activity(
     target_address: AnyLinkableHash,
 ) -> ExternResult<ValidateCallbackResult> {
     // Check the entry type for the given action hash
-    let agent_pk = target_address
-        .into_agent_pub_key()
-        .ok_or(wasm_error!(WasmErrorInner::Guest(
-            "Invalid target address".to_string()
-        )))?;
+    let agent_pk =
+        target_address
+            .into_agent_pub_key()
+            .ok_or(wasm_error!(WasmErrorInner::Guest(
+                "Invalid target address".to_string()
+            )))?;
     let action_hash = base_address
         .into_action_hash()
         .ok_or(wasm_error!(WasmErrorInner::Guest(
             "No action hash associated with link".to_string()
         )))?;
-    let _ = must_get_agent_activity(agent_pk, ChainFilter::new(action_hash))?;
+    let e = must_get_agent_activity(agent_pk, ChainFilter::new(action_hash));
+    tracing::info!("e is {e:?}");
```

Comment:
Tracing needs to be kept?

Reply:
I'm debugging something. This PR is not yet ready. I'll post it in the PR channel once I've fixed the zero arc test.